PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -77,10 +77,16 @@ - (void)setOptions:(NSDictionary *)options forObject:(id)object animated:(BOOL)a
                     [self setValue:newOptions[key] forKey:[key stringByAppendingFormat:@"For%@WithJSON", [object class]]];
```

Comment:
This failed because we have no `*ForPSPDFImageDocumentWithJSON:`. So instead of implementing methods for image documents which call the PSPDFDocument methods, we try to see if the super class has a method implemented which can call before using KVO.